src/alchemtest/tests/test_gmx.py

```diff
@@ -0,0 +1,33 @@
+'''Tests for all the gromacs dataset'''
+import os
+import pytest
+
+from alchemtest.gmx import load_benzene
+
+class Testbenzene():
```

Comment:
No parentheses should be safe for py2/py3 --- it will be old-style class for py 2 but I don't think we care. The alternative `TestBenzene(object)` could be used but is clunky for py 3. Mainly I wanted the name to be camel-case:
```suggestion
class TestBenzene:
```